WebAPI/Endpoints/LibrariesMinimalApi.cs

```diff
@@ -1,5 +1,4 @@
-﻿using Amazon.Util;
-using Application;
+﻿using Application;
```

Comment:
**Suggestion:** The `Amazon.Util` namespace has been removed. If this namespace was being used in the file, it might cause a compile-time error. Please ensure that removing this namespace does not affect the functionality of the code. [possible issue]
```suggestion
﻿using Application;
using Amazon.Util;
```